x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -107,10 +107,7 @@ const ActionsComponent: React.FC<ActionProps> = ({
   const isContextMenuDisabled = useMemo(() => {
     return (
       eventType !== 'signal' &&
-      !(
```

Comment:
Hey @dasansol92, was it necessary to remove the `ecsData.event?.kind?.includes('alert')` check? I don't think the behavior should change if that's also there, unless I'm missing something. This change _may_ cause issues as this table is also being used in the alert page. Will test it to check it out, but might the below work?

```
const isEndpointEvent = (ecsData.event?.kind?.includes('event') || ecsData.event?.kind?.includes('alert')) && ecsData.agent?.type?.includes('endpoint');

return eventType !== 'signal' && !isEndpointEvent;
 ````

Reply:
Was able to pull it down and test it, and it looks like this does break the event filters in the alert page view of the table.  Updating this here to this and the other change in `alert_context_menu` should make it functional in both parts of the app
```
  const isContextMenuDisabled = useMemo(() => {
    const endpointEventKinds = ['event', 'signal', 'alert'];
    const isEndpointEvent =
      endpointEventKinds.some((eventKind) => ecsData.event?.kind?.indexOf(eventKind) !== -1) &&
      ecsData.agent?.type?.includes('endpoint');
    return eventType !== 'signal' && !isEndpointEvent;
  }, [ecsData, eventType]);
```

Reply:
Yeah, what we want is to hide the `Add endpoint event filter` option if the event is not an `event` (is an `alert` or whatever else) and if the `agent.type` is not `endpoint`. So this option (`Add endpoint event filter`) needs to be displayed only for:
`agent.type` === 'endpoint` && `event.kind` === `event`